.github/workflows/esti.yaml

```diff
@@ -295,6 +295,12 @@ jobs:
           AWS_SECRET_ACCESS_KEY: minio123
           STORAGE_NAMESPACE: s3://test-bucket/data
           REPOSITORY: lakefsfs-contract-test
+
+      - name: Build current lakeFS SDK
+        # TODO(ariels): Remove this in favour of a single lakeFS client version
```

Comment:
are we removing this or the legacy version?

Reply:
Both!  For now...

Later on we will remove the old-style API.  But we need to allow our users to upgrade servers first, then clients.